consensus/src/epoch_manager.rs

```diff
@@ -951,7 +956,7 @@ impl EpochManager {
             UnverifiedEvent::BatchMsg(_)
             | UnverifiedEvent::SignedBatchInfo(_)
             | UnverifiedEvent::ProofOfStoreMsg(_) => {
-                if self.quorum_store_enabled {
+                if self.quorum_store_enabled || self.recovery_mode {
```

Comment:
Hmm, I think we need to filter out the quorum store events on recovery, just without printing out error messages. Quorum store can't do anything productive right now when in recovery.

Reply:
Made the change. Now the filter_quorum_store_events method returns Result<bool,Err>. The result indicates whether to filter out the messages or not.